app/models/song.rb

```diff
@@ -0,0 +1,3 @@
+class Song < ApplicationRecord
+    belongs_to :user
```

Comment:
This indent is 4 spaces. Ruby convention is to use 2. Check your IDE to ensure your settings are accurate.